### ISSUE TITLE
[Chore]: Create monorepo

### ISSUE THREAD

**Issue by andriishupta** [Developer-DAO/developerdao.com#266]
### Context / Background / How This Idea Came To Fruition

Connect CMS and FE application to turborepo's monorepo- https://turborepo.org/

Also, add the correct build pipeline to use on Site and CMS deployments

References https://github.com/Developer-DAO/cms/issues/4

### Problem / Opportunity

To develop new functionality, Site developers would need to clone CMS from separate repo

### Proposed Solution / How It Can Be Improved

Keep up with trends and make life easier for Site development

### Testing the changes

Manual

**Comment by kempsterrrr** [comment]
If someone is up for it I'd love us to move forward with this, adding the repo @mhmulford0 shared in discord for reference - https://github.com/mhmulford0/dd-website-monorepo

I'm not sure how easy/hard this so don't want to push someone to do this when I don't have time though!

When we do it we should try to maintain the commit history across the two repo's if we cam

**Comment by mhmulford0** [comment]
yeah i'm thinking monorepo first then turbo repo. i just now got the commit history onto the monorepo.
